UnitTests.Deviationlark/PhoneBookTests/UnitTest1.cs

```diff
@@ -0,0 +1,50 @@
+using PhoneBook;
+namespace PhoneBookTests;
+
+[TestClass]
+public class PhoneBookTest
+{
+    [TestMethod]
+    [DataRow("+38978333333")]
+    public void ValidateNumber_IsValidNumber_ReturnTrue(string phoneNumber)
+    {
+        var result = PhoneBookValidation.ValidateNumber(phoneNumber);
+        Assert.IsTrue(result);
+    }
+    [TestMethod]
+    [DataRow("389783333333")]
+    [DataRow("")]
+    public void ValidateNumber_IsInvalidNumber_ReturnFalse(string phoneNumber)
```

Comment:
IsInvalidNumber  ....> yes it is invalidnumber....>why return false??
Name is inconsistent.